pyrasterframes/src/main/python/docs/aggregation.pymd

```diff
@@ -10,13 +10,65 @@ import os
 spark = create_rf_spark_session()
 ```
 
-## Cell Counts & Tile Mean
+There are 3 types of aggregate functions: tile aggregate, DataFrame aggregate, and element-wise local aggregate. In the @ref:[`tile aggregate functions`](reference.md#tile-statistics), we are computing a statistical summary per row of a `tile` column in a DataFrame. In the @ref:[`DataFrame aggregate functions`](reference.md#aggregate-tile-statistics), we are computing statistical summaries over all of the cell values *and* across all of the rows in the DataFrame or group. In the  @ref:[`element-wise local aggregate functions`](reference.md#tile-local-aggregate-statistics), we are computing the element-wise statistical summary across a DataFrame or group of tiles.
```

Comment:
i think maybe the backticks around "tile aggregate functions" and the other two are a bit much... maybe italics to indicate that we are introducing a bit of jargon.